config/initializers/app_artifacts.rb

```diff
@@ -1,11 +1,12 @@
 require 'app_artifacts'
 
 AppArtifacts.setup do |store|
-  # When adding or removing certs, make sure to update the 'saml_endpoint_configs' config
-  store.add_artifact(:saml_2021_cert, '/%<env>s/saml2021.crt')
-  store.add_artifact(:saml_2021_key, '/%<env>s/saml2021.key.enc')
-  store.add_artifact(:saml_2022_cert, '/%<env>s/saml2022.crt')
-  store.add_artifact(:saml_2022_key, '/%<env>s/saml2022.key.enc')
+  years = IdentityConfig.store.saml_endpoint_configs.map { |s| s[:suffix].to_i }
```

Comment:
Do we care that they're numeric if we're just concatenating them into strings?

```suggestion
  years = IdentityConfig.store.saml_endpoint_configs.pluck(:suffix)
```

Reply:
I guess we don't care! good idea